static/sass/styles.scss

```
@@ -75,6 +72,11 @@ $breakpoint-navigation-threshold: 900px;
   }
 }
 
+body:not(.docs) .p-navigation__banner {
```

I opened an issue to address that on Vanilla side, please add a comment for reference:
https://github.com/canonical/vanilla-framework/issues/4947

```suggestion
// TODO: to be removed when fixed in Vanilla
// https://github.com/canonical/vanilla-framework/issues/4947
body:not(.docs) .p-navigation__banner {
```